src/textures/CompressedArrayTexture.js

```diff
@@ -0,0 +1,18 @@
+import { ClampToEdgeWrapping } from '../constants.js';
+import { CompressedTexture } from './CompressedTexture.js';
+
+class CompressedArrayTexture extends CompressedTexture {
+
+	constructor( depth, mipmaps, width, height, format, type, mapping, wrapS, wrapT, magFilter, minFilter, anisotropy, encoding ) {
+
+		super( mipmaps, width, height, format, type, mapping, wrapS, wrapT, magFilter, minFilter, anisotropy, encoding );
+
+		this.isDataArrayTexture = true;
```

Comment:
Can we make this work with `this.isCompressedArrayTexture = true;` instead? I think it will cause confusion later if we misrepresent the type here.